firefox-ios/Client/Telemetry/TelemetryWrapper.swift

```diff
@@ -83,86 +88,7 @@ class TelemetryWrapper: TelemetryWrapperProtocol, FeatureFlaggable {
         migratePathComponentInDocumentsDirectory("MozTelemetry-Default-mobile-event", to: .cachesDirectory)
         migratePathComponentInDocumentsDirectory("eventArray-MozTelemetry-Default-mobile-event.json", to: .cachesDirectory)
```

Comment:
btw this _probably_ can go now, potentially ensuring that any data left over in the `cachesDirectory` from this can be deleted to. But this can be done in a followup.